swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphParameter.qll

```diff
@@ -0,0 +1,69 @@
+import swift
```

Comment:
It would be good to have a comment explaining what this file is.  It appears to create the `CfgImpl` module from the shared `codeql.controlflow.Cfg` library, via an input signature that is also defined here.

Reply:
I'm not really happy with the name either. I'm tempted to merge the file with ControlFlowGraphImplSpecific.

Reply:
I think merging this file with `ControlFlowGraphImplSpecific` is a great idea!